src/cast_core/src/actors/RunCommand.cpp

```diff
@@ -28,11 +26,76 @@
 
 #include <boost/log/trivial.hpp>
 
+#include <cast_core/actors/RunCommand.hpp>
+
 #include <gennylib/ExecutionStrategy.hpp>
 #include <gennylib/MongoException.hpp>
 #include <gennylib/v1/RateLimiter.hpp>
 #include <gennylib/value_generators.hpp>
 
+namespace {
```

Comment:
If you put this functionality into a widget in the `v1` namespace, you can run it in `test_gennylib_with_server` without having to construct an actor.

Reply:
ya..considered it..but I decided to just leave as-is the logic is simple enough to test with the Actor itself